frontend/src/components/Topbar.jsx

```
@@ -69,9 +69,24 @@ export const Topbar = () => {
       <div className={styles.container}>
-        <div className={styles.logo} onClick={() => navigate('/dashboard')}>STUDIFY</div>
+        <button
+          className={styles.logo}
+          onClick={() => navigate("/dashboard")}
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              e.preventDefault();
+              navigate("/dashboard");
+            }
+          }}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)